### PR TITLE
Scripts/SQL: Added some gear(sets) mods

### PR DESCRIPTION
**Pull request by Hozu**
-Gearsets: Bowman's set, Altdorf's/Wilhelm's earring, Helenus's/Cassandra's earring, Lava's/Kusha's Ring (for the last two I removed them from scripts/globals/items because they don't belong there), and Iron Ram Hauberk set. Source for Bowman's set: http://ffxi.allakhazam.com/forum.html?forum=252&mid=1231495866143984605
-Added a modifier for enhances Stoneskin "HP" absorbed and gave it to the four items that were previously hardcoded in the script.
-Added 3% Snapshot to Zha'Go's Barbut https://www.bg-wiki.com/bg/Zha'Go's_Barbut
-Added +10% Ninjutsu damage to Yagentoshiro. http://ffxiclopedia.wikia.com/wiki/Yagentoshiro

### CODE REVIEW

**Comment by TeoTwawki** [comment]
> for the last two I removed them from scripts/globals/items because they don't belong there

Bless you kind sir.

File: scripts/globals/gear_sets.lua [hunk 1]
```diff
@@ -102,7 +102,10 @@ local GearSets =  {
              {id = 84, items = {11867,10868,10865}, matches = 2, matchType = matchtype.any, mods = {{MOD_REFRESH, 3, 0, 0}} }, -- Heka's body + NQ or HQ Khat = 3 tick refresh
              {id = 85, items = {10868,11870,11864,10865}, matches = 2, matchType = matchtype.any, mods = {{MOD_REFRESH, 2, 0, 0}} }, -- Nefer body/head NQ/HQ combo gives Refresh +2
              {id = 86, items = {15852,15853}, matches = 2, matchType = matchtype.any, mods = {{MOD_HP, 50, 0, 0},{MOD_MP, 50, 0, 0}} }, -- Dasra's/Nasatya's Ring set gives HP/MP +50
-
+             {id = 87, items = {16037,16038}, matches = 2, matchType = matchtype.any, mods = {{MOD_MATT, 5, 0, 0},{MOD_MACC, 5, 0, 0}} }, -- Helenus's/Cassandra's earring set: Mag atk bonus+5 and Mag acc +5
```
Comment (xdemolish) [inline scripts/globals/gear_sets.lua]: id should be 88 (next one 90 since there are two mods in this)

File: scripts/globals/gear_sets.lua [hunk 2]
```diff
@@ -102,7 +102,10 @@ local GearSets =  {
...
              {id = 85, items = {10868,11870,11864,10865}, matches = 2, matchType = matchtype.any, mods = {{MOD_REFRESH, 2, 0, 0}} }, -- Nefer body/head NQ/HQ combo gives Refresh +2
              {id = 86, items = {15852,15853}, matches = 2, matchType = matchtype.any, mods = {{MOD_HP, 50, 0, 0},{MOD_MP, 50, 0, 0}} }, -- Dasra's/Nasatya's Ring set gives HP/MP +50
-
+             {id = 87, items = {16037,16038}, matches = 2, matchType = matchtype.any, mods = {{MOD_MATT, 5, 0, 0},{MOD_MACC, 5, 0, 0}} }, -- Helenus's/Cassandra's earring set: Mag atk bonus+5 and Mag acc +5
+             {id = 88, items = {15850,15851}, matches = 2, matchType = matchtype.any, mods = {{MOD_ATT, 6, 0, 0},{MOD_ACC, 12, 0, 0},{MOD_DEF, 6, 0, 0}} }, -- Lava's/Kusha's earring set: Atk+6/Acc+12
+             {id = 91, items = {16146,14588,15009,16315,15755},  matches = 2, matchType = matchtype.any, mods = {{MOD_FIRERES, 5, 5, 10},{MOD_ICERES, 5, 5, 10},{MOD_WINDRES, 5, 5, 10},{MOD_EARTHRES, 5, 5, 10},{MOD_THUNDERRES, 5, 5, 10},{MOD_WATERRES, 5, 5, 10},{MOD_LIGHTRES, 5, 5, 10},{MOD_DARKRES, 5, 5, 10}} }, --  Iron Ram Haubert Set
```
Comment (xdemolish) [inline scripts/globals/gear_sets.lua]: id 93

File: scripts/globals/gear_sets.lua [hunk 3]
```diff
@@ -102,7 +102,10 @@ local GearSets =  {
...
              {id = 86, items = {15852,15853}, matches = 2, matchType = matchtype.any, mods = {{MOD_HP, 50, 0, 0},{MOD_MP, 50, 0, 0}} }, -- Dasra's/Nasatya's Ring set gives HP/MP +50
-
+             {id = 87, items = {16037,16038}, matches = 2, matchType = matchtype.any, mods = {{MOD_MATT, 5, 0, 0},{MOD_MACC, 5, 0, 0}} }, -- Helenus's/Cassandra's earring set: Mag atk bonus+5 and Mag acc +5
+             {id = 88, items = {15850,15851}, matches = 2, matchType = matchtype.any, mods = {{MOD_ATT, 6, 0, 0},{MOD_ACC, 12, 0, 0},{MOD_DEF, 6, 0, 0}} }, -- Lava's/Kusha's earring set: Atk+6/Acc+12
+             {id = 91, items = {16146,14588,15009,16315,15755},  matches = 2, matchType = matchtype.any, mods = {{MOD_FIRERES, 5, 5, 10},{MOD_ICERES, 5, 5, 10},{MOD_WINDRES, 5, 5, 10},{MOD_EARTHRES, 5, 5, 10},{MOD_THUNDERRES, 5, 5, 10},{MOD_WATERRES, 5, 5, 10},{MOD_LIGHTRES, 5, 5, 10},{MOD_DARKRES, 5, 5, 10}} }, --  Iron Ram Haubert Set
+             {id = 99, items = {16035,16036}, matches = 2, matchType = matchtype.any, mods = {{MOD_AGI, 8, 0, 0}} } -- Altdorf's/Wilhelm's earring: AGI+8
```
Comment (xdemolish) [inline scripts/globals/gear_sets.lua]: id 101

**Comment by Hozu** [comment]
Shit you're right I didn't notice it because I saw the damage go up from spells, so I just assumed mag acc was working too.

**Pull request merged**